src/librawspeed/decoders/NefDecoder.cpp

```diff
@@ -115,13 +115,13 @@ RawImage NefDecoder::decodeRawInternal() {
 
   mRaw->dim = iPoint2D(width, height);
 
-  raw = mRootIFD->getIFDWithTag(static_cast<TiffTag>(0x8c));
-
-  const TiffEntry* meta;
-  if (raw->hasEntry(static_cast<TiffTag>(0x96))) {
-    meta = raw->getEntry(static_cast<TiffTag>(0x96));
```

Comment:
Uh, what was the old code even trying to do?
Looking for an IFD with one specific tag, but then opting for completely another tag...

Reply:
Yep, definitely not entirely correct.

Reply:
Turns out, we don't need the IFD at all, so maybe better to just recursively search for the desired 0x96 and 0x8c tags (`getIFDsWithTag`/`getIFDWithTag` don't do anything different to that recursive tag search anyway).